### PR TITLE
Complete: tasks705-708 from MRQA and hasPart KB datasets

### PR DESCRIPTION
**Pull request by Maitreyapatel**
Please review these tasks.

### CODE REVIEW

**Comment by swarooprm** [comment]
1. task705: "Given a question and context passage"--> "Given a question and a context passage", the best answer may be undefined. Check if the answer is always a shortest continous span or something like that, in that case you can mention it in definition. 
2. task706: "Given the context passage, generate the question such that the possible answers"-->"Given a context passage, generate a question such that its answer". 
3. task707: "Given an entity as input, give the answer having another entity which is the part of the input entity. These are the entities of the meronym."-->"Given an entity as input, output another entity which is part of the input entity. These are entities of meronym."
4. task708, "These are the entities of the meronym"-->"These are entities of meronym". The sentence is not true for this task 'In simpler terms, a meronym (i.e., output entity) is in a part-of relationship with its holonym (i.e., input entity).". You can replace input and output entity by 1st, 2nd entity or something in that line. Make sure the classes are balanced in this classification tasks, report the class count.

I like that task705 instances have a lot of variety in terms of input length and question format (sometimes its fill in the blank type).
Overall, these are minor changes in language, you should be able to fix it quickly. @Maitreyapatel

**Comment by Maitreyapatel** [comment]
Thank you for reviewing the tasks! Please refer following new changes:
1. task705: Minor language change and added new criteria where the answer is shortest possible span from the passage.
2. task706: Minor language change. 
3. task707: Minor language change.
4. task708: Changed the input/output entities as first/second entities, respectively in the definition. Also, the instances are completely balanced (i.e., both classes have 3000 instances).

**Comment by swarooprm** [comment]
I am going to accept your tasks, you have done a great job. Your HW2 is complete now.

Would you like to submit the same tasks to our big project repo https://github.com/allenai/natural-instructions-expansion/pulls? You just need to fork that repo, rename tasks starting from the latest number in the repo (which at this point is 469) and create PR there. This way, you can be part of our research on "learning from instructions instead of learning from examples (conventional machine learning)". This is purely optional and your HW2 grade does not depend on this. @Maitreyapatel

**Comment by Maitreyapatel** [comment]
Thanks for accepting my tasks. I will add these tasks to your project repo.
